services/shhext/mailservers/peerstore.go

```diff
@@ -0,0 +1,67 @@
+package mailservers
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/p2p/enode"
+)
+
+var (
+	// ErrNoConnected returned when mail servers are not connected.
+	ErrNoConnected = errors.New("no connected mail servers")
+)
+
+// PeersProvider is an interface for requesting peers.
```

Comment:
```suggestion
// PeersProvider is an interface for requesting list of peers.
```
To make it more explicit that we're requesting a list of existing peers, and not new peers.